### PR TITLE
updating to ghostnet for all specs that need it

### PR DESCRIPTION
**Pull request by alexzbusko**

### CODE REVIEW

File: tests/e2e/data/config-taquito-test-environment-invalid-initial-storage-string.json [hunk 1]
```diff
@@ -32,33 +32,33 @@
...
     },
     "network": {
-        "ithacanet": {
-            "label": "ithacanet",
-            "rpcUrl": "https://ithacanet.ecadinfra.com",
-            "protocol": "Psithaca2MLRFYargivpo7YvUr7wUDqyxrdhC5CQq78mRvimz6A",
+        "ghostnet": {
+            "label": "ghostnet",
+            "rpcUrl": "https://rpc.ithacanet.teztnets.xyz",
+            "protocol": "PtJakart2xVj7pYXJBXrqHgd82rdkLey5ZeeGwDgPp9rhQUbSqY",
```
Comment (hu3man) [inline tests/e2e/data/config-taquito-test-environment-invalid-initial-storage-string.json:38]: This is fine for now but given that ghostnet will not have a stable protocol, we may want to consider how we can dynamically handle protocol upgrades on ghostnet moving forward

**Review by hu3man — APPROVED** [review]
LGTM 🚀

**Review by jchenche — APPROVED** [review]

File: tests/e2e/data/config-taquito-test-environment-invalid-initial-storage-string.json (hunk 1 above)
Comment (alexzbusko) [reply tests/e2e/data/config-taquito-test-environment-invalid-initial-storage-string.json:38]: Yeah we can query teztnets to see what is on the about page for ghostnet and just take the highest block protocol. The thing is these files are JSON and we may have to build these files if we want anything like that. That isn't a trivial thing to do so I am not going to do it now/here

File: tests/e2e/taqueria-plugin-flextesa.spec.ts [hunk 2]
```diff
@@ -407,7 +407,7 @@ describe('E2E Testing for taqueria flextesa plugin sandbox starts/stops', () =>
 	// TODO: Currently it cannot be done until the output will be places to stdout
 	// Issue to implement the test: https://github.com/ecadlabs/taqueria/issues/368
```
[the remark this answers is not in the document]
Comment (alexzbusko) [reply tests/e2e/taqueria-plugin-flextesa.spec.ts:408]: Yeah I will remove the comment

**Pull request merged**